### DISCUSSION TITLE
abp 4.3.0  ef core migration empty up() and down()

### DISCUSSION THREAD

**Discussion by maxwelltsai1536** [abpframework/abp#9037] [Q&A] [answered]
when i add model and use add-migration addModel , the migration file have empty up() and down()
i wirted DbSet<Model> in DbContext,y ef core cant get change

**Comment by maliming** [comment, marked as answer]
You can refer to the Todu sample.

https://github.com/abpframework/abp-samples/tree/master/TodoApp

**Reply by maxwelltsai1536** [reply]
idk what problem with this case,but i used by apb cli created project,actually i want to know y im wrong

**Comment by maxwelltsai1536** [comment]
i found this problem is i havent config in OnModelCreating,then abp dont think this is new change,when i configed table name in OnModelCreating ,then i got the new change up() and down()

**Comment by maxwelltsai1536** [comment]
but y in abp have this setting,is wired,because i just use ef core default setting that's wrong? or this is a bug?
